src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
@rustd Weighing in.
The name you enter in the portal should be the name you use to access the connection string. 

After aspnet/Configuration#376, the only way to accommodate the `Data` section in the templates is have customers type `Data:MyConnection:ConnectionString` into the portal, which is just **awful**.

Unless I am missing context from why previously agreed to the `data` section, we should merge these changes in.